package/yast2-ruby-bindings.spec

```diff
@@ -28,21 +28,23 @@ BuildRequires:  cmake
 BuildRequires:  gcc-c++
 BuildRequires:  yast2-core-devel
 BuildRequires:  yast2-devtools >= 3.1.10
-%if 0%{?suse_version} == 1310
-BuildRequires:  rubygem-fast_gettext < 3.0
-BuildRequires:  rubygem-rspec
-Requires:       rubygem-fast_gettext < 3.0
-%else
 BuildRequires:  rubygem(%{rb_default_ruby_abi}:fast_gettext) < 3.0
 BuildRequires:  rubygem(%{rb_default_ruby_abi}:rspec)
 Requires:       rubygem(%{rb_default_ruby_abi}:fast_gettext) < 3.0
-%endif
 BuildRequires:  ruby-devel
 Requires:       yast2-core >= 3.2.2
 BuildRequires:  yast2-core-devel >= 3.2.2
 # MenuBar-shortcuts-test.rb
 Requires:       yast2-ycp-ui-bindings       >= 4.3.7
 BuildRequires:  yast2-ycp-ui-bindings-devel >= 4.3.7
+%ifarch s390 s390x
+# s390 specific frame title that is read from readvalues from s390-tools
+# needed also for tests, so build require it
+BuildRequires:  s390-tools
```

Comment:
Alternatively we could fix the [read_values](https://github.com/yast/yast-ruby-bindings/blob/a66796909dfa0ab42642e9969244622f08ca8b02/src/ruby/yast/y2start_helpers.rb#L125) method to handle missing `read_values` binary.

The advantage would be smaller build dependencies, during test we do not care much about the machine title...

But I guess the `s390-tools` package is small and it is an acceptable dependency.

Reply:
yeap and also quite mandatory on s390. It quite surprise me that it is not already installed in build env.
And I prefer to not touch production code for test cases. Only if we want to handle missing s390-tools also outside of test, but there I prefer consistency with having title always.